help/en/releasenotes/current-draft-note.shtml

```diff
@@ -439,7 +439,10 @@
     <h3>Operations</h3>
         <a id="Operations" name="Operations"></a>
         <ul>
-            <li></li>
+            <li>Improvements to the car, locomotive, and train preview and print tools.</li>
+            <li>Minor improvement to the train build report when routing.</li>
+            <li>To see all changes to OperationsPro, use the link below</li>
+            <li><a href="https://github.com/pulls?q=is%3Apr+author%3ADanielBoudreau+archived%3Afalse+is%3Aclosed">OperationsPro Changes</a></li>
```

Comment:
I suggest changing `is%3Aclosed` to `is%3Amerged`. Maybe it doesn't matter for you, but I have a number of PRs that I for different reasons closed instead of merged. By having `is%3Amerged`, you get only those PRs that are actually merged.

https://github.com/pulls?q=is%3Apr+author%3ADanielBoudreau+archived%3Afalse+is%3Amerged

Reply:
One and the same for me.  But I will use merged next time. Thanks.